.gitignore

```diff
@@ -159,6 +159,8 @@ cython_debug/
 #  option (not recommended) you can uncomment the following to ignore the entire idea folder.
 #.idea/
 
+.DS_Store
```

Comment:
Historically, we've been pretty strict as a team with ensuring that any local env type ignores are handled in your local and not here.  See: https://github.com/GSA/search-gov/blob/main/.gitignore#L3-L5. I don't consider this a blocker by any means, but if we're trying to keep parity with the other existing repos, we should start this one out adhering to the same standard.

Reply:
All of that makes sense to me, thank you for the review! I updated the gitignore file